sdk/nodejs/automation/errors.ts

```diff
@@ -27,7 +28,8 @@ export class CommandError extends Error {
 }
 
 /**
- * ConcurrentUpdateError is thrown when attempting to update a stack that already has an update in progress.
+ * An error thrown when attempting to update a stack that already has an updated
```

Comment:
```suggestion
 * An error thrown when attempting to update a stack that already has an update
```

Reply:
My non-native English brain thinks `[...] update a stack that is already being updated` or `[...] update a stack that is undergoing an update operation`

Reply:
Great suggestions! Incorporated :+1: